<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnLeft">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>opna</string>
    </property>
   </widget>
   <widget class="QLabel" name="picL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>300</width>
      <height>240</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Impath">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>641</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Slóð</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusL">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>60</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>vinstri mynd</string>
    </property>
   </widget>
   <widget class="QLabel" name="picR">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>300</width>
      <height>240</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="statusR">
    <property name="geometry">
     <rect>
      <x>636</x>
      <y>60</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>hægri mynd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProcess">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>vinna mynd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDepthmap">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>dýptarmynd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPCL">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PCL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFull">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fullrun</string>
    </property>
   </widget>
   <widget class="QLabel" name="output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>831</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Skilaboð</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>stereo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>loka PCL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>560</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>loka forriti</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenLeft">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>opna</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImpathL">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>641</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Slóð</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenRight">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>opna</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>óunnin mynd</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>vinstri mynd</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>hægri mynd</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImpathR">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>641</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Slóð</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>460</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>dýptarmynd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenDisp">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>opna</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImpathDisp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>641</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Slóð</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>nrDisparitys</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dispNrText">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>500</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>500</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>minDisparity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minDispText">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P2Text">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>530</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>530</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="P1Text">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>530</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="speckleSizeText">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>500</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>speckleSize</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="speckleRangeText">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>530</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>530</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>specleRange</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>uniqueness</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>530</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>disp12MaxDiff</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="disp12Text">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="uniqueText">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>500</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1311</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
